source/uwp/Renderer/lib/AdaptiveElementParserRegistration.cpp

```diff
@@ -33,24 +33,22 @@ using namespace ABI::Windows::UI;
 
 namespace AdaptiveNamespace
 {
+    const char* c_upwElementParserRegistration = "447C3D76-CAAD-405F-B929-E3201F1537AB";
```

Comment:
>const char* [](start = 4, length = 11)

instead of declaring in the header and defining here, should it be a `constexpr` in the header?

Reply:
Indeed. Updated.

---
In reply to: [322488612](https://github.com/microsoft/AdaptiveCards/pull/3429#discussion_r322488612) [](ancestors = 322488612)